docs/SPIR-V.rst

```diff
@@ -358,6 +358,21 @@ interface variables:
   main([[vk::location(N)]] float4 input: A) : B
   { ... }
 
+Macro for SPIR-V
+----------------
+
+If SPIR-V CodeGen is enabled and ``-spirv`` flag is used as one of the command
+line options (meaning that "generates SPIR-V code"), it defines an implicit
+macro ``__spirv``. For example, this macro definition can be used for SPIR-V
+specific part of the HLSL code:
+
+.. code:: hlsl
+
+  #ifdef __spirv
```

Comment:
TODOs based on offline discussion:
* Move it to https://github.com/microsoft/DirectXShaderCompiler/tree/master/tools/clang/test/HLSLFileCheck/hlsl/preprocessor/version_defines 
* Change `__spirv` to `__SPIRV`